### ISSUE TITLE
Cubejs s3 Iam role

### ISSUE THREAD

**Issue by ShreenidhiVenkatesan** [cube-js/cube#6795]
**Problem**

How to associate aws s3 bucket for storing pre-aggregation and cube store by associating ec2 and role
Without providing access key and secret key.

**Comment by ShreenidhiVenkatesan** [comment]
How to make use of instance role while using docker compose to set up cube in aws ec2

**Comment by iamsameervaidya** [comment]
Yeah - same problem here - the code won't initialize without the CUBESTORE_AWS_SECRET_ACCESS_KEY_ID and CUBESTORE_AWS_SECRET_ACCESS_KEY env vars set - and our company has standardized on EC2 roles and prefer not to use the S3 Keys and Secrets in source code

**Comment by bart-sofomo** [comment]
Hello
any eta on that?

**Comment by igorlukanin** [comment]
No ETA as of now. We're open to community contributions though.
